iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/OperatingStatus.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.Identifiable;
+
+import java.util.Objects;
+
+/**
+ * `Identifiable` is needed as generic type because we also have to support HVDC lines which are not `Connectable`
```

Comment:
It might be interesting to have a description of what the extensions is for, even if in that case it is easy to understand.